Dockerfile

```diff
@@ -20,4 +21,10 @@ RUN apt-get update && apt-get install -y python3 python3-pip python-dev libmupdf
 
```

Comment:
The final image size here is over 650MB, I think we can get that down a bit.

Instead of building the final image off of `ubuntu`, we could consider using the `python:<version>-slim` or `python:<version>-alpine` base images instead.

The `python:<version>-alpine` base images specifically are based on Alpine Linux and are obscenely small (<5MB) before the relevant dependencies are installed, but I had trouble getting it running quickly due to the `mupdf` dependency not building with `apk add mupdf`.

That being said, I tested this locally (switching the base image to `python:3.9-slim` and removing the Python depending in the `apt-get install`, and got a built image 347MB in size.

Reply:
Yeah good shout. 

I had issues getting the built `pdfalto` binary running on `python:*` and meant to circle back and revisit but slipped my mind.

Reply:
Switched `build` image to debian rather than ubuntu and this resolved issues I was seeing. Final image now 279MB